scenarios/node_hog.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: litmuschaos.io/v1alpha1
+description:
+  message: |
+    Give a cpu spike on a node belonging to a deployment
+kind: ChaosExperiment
+metadata:
+  name: node-cpu-hog
+spec:
+  definition:
+    scope: Cluster
+    permissions:
+      - apiGroups:
+          - ""
+          - "batch"
+          - "apps"
+          - "litmuschaos.io"
+        resources:
+          - "jobs"
+          - "pods"
+          - "pods/log"
+          - "events"
+          - "chaosengines"
+          - "chaosexperiments"
+          - "chaosresults"
+        verbs:
+          - "create"
+          - "list"
+          - "get"
+          - "patch"
+          - "update"
+          - "delete"
+      - apiGroups:
+          - ""
+        resources:
+          - "nodes"
+        verbs:
+          - "get"
+          - "list"
+    image: "litmuschaos/go-runner:1.8.2"
```

Comment:
It would be better to use the experiments from the github, because this version may outdate and not equal to version of litmus installed. So maybe would be better to install experiments with litmus from: https://hub.litmuschaos.io/api/chaos/1.9.0?file=charts/generic/experiments.yaml

Reply:
Took a look at this yesterday. I agree that this would be better to install the experiments from the versioned url's. 

From my understanding I have to edit the experiment once I apply it to give it the correct node name for this specific experiment I have been working on. Is that correct?

Reply:
the node name can be provided via `APP_NODE` env var in the ChaosEngine resource
See an example of the ChaosEngine that provides it (with blank value in the example): https://docs.litmuschaos.io/docs/node-cpu-hog/#sample-chaosengine-manifest

Once you set it via the ChaosEngine, it will override the value in the ChaosExperiment and be used.
That will allow you to rely on the formal released experiments which is aligned with litmus operator version.